jobs/atc/templates/generic_oauth_ca_cert.erb

```diff
@@ -0,0 +1 @@
+<%= p("generic_oauth.ca_cert", "") %>
```

Comment:
dont need "" here if default is set in spec.

Reply:
Thanks.  Removed "" from spec.